main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        ("Western",),
+        ("Action",),
+        ("Drama",),
+    ]
```

Comment:
According to the checklist, you should use a list of tuples for better iterating. In your code, you are using a list of tuples with only one element. It's better to have a list of strings instead. This way you can avoid unnecessary unpacking in the loop.